Doc/library/asyncio-stream.rst

```diff
@@ -87,12 +69,8 @@ and work with streams:
 
       The *ssl_handshake_timeout* parameter.
 
-   .. deprecated-removed:: 3.8 3.10
-
-      `open_connection()` is deprecated in favor of :func:`connect`.
-
 .. coroutinefunction:: start_server(client_connected_cb, host=None, \
-                          port=None, \*, loop=None, limit=2**16, \
+                          port=None, \*, loop=None, limit=None, \
```

Comment:
The constant `_DEFAULT_LIMIT` is still being used as the default for *limit* in `start_server()`, so this can be restored.
```suggestion
                          port=None, \*, loop=None, limit=2**16, \
```

Reply:
I can do this in a separate PR if needed. This change is rather low priority in comparison to reverting the API.

Reply:
Sure, although right now docs are the same as in 3.7. So I'm not sure it's needed tbh.

Reply:
> So I'm not sure it's needed tbh.

Okay, I won't worry about it then, it's fairly trivial.